### DISCUSSION TITLE
How to run cypress scripts headed but without GUI runner

### DISCUSSION THREAD

**Discussion by noordeen123** [cypress-io/cypress#30785] [Questions and Help] [answered]
I want to run the scripts in headed mode without the Test Runner GUI using the command npx cypress run --headed.
is there any way to do so?

**Comment by jennifer-shehane** [comment, marked as answer]
Use `cypress run --headed --no-runner-ui`  this shows the real browser but hides the Cypress Command Log UI / reporter chrome.

`--no-runner-ui` removes the runner overlay while keeping a headed browser. https://on.cypress.io/command-line#cypress-run-no-runner-ui
